test/now.js

```diff
@@ -54,3 +54,38 @@ test("now should be up to date when ticker is reactivated, #271", (done) => {
         done()
     }, 200)
 })
+
+describe("given desynchronization is enabled", () => {
+    let actual
+
+    beforeEach(() => {
+        jest.useFakeTimers("modern")
+        jest.setSystemTime(new Date("2015-10-21T07:28:00Z"))
+
+        utils.desynchronizeNowForTests()
```

Comment:
commenting out this line did not cause the test to fail, even fixing the above bug.

See also comment on original bug.